plugins/slick.rowdetailview.js

```diff
@@ -572,6 +573,7 @@
         width: _options.width,
         resizable: false,
         sortable: false,
+        alwaysRenderColumn: _options.alwaysRenderColumn,
```

Comment:
so I assume that if user is not passing it, the ideal input should be `false`, then it's better to define it as such
`alwaysRenderColumn: _options.alwaysRenderColumn || false,`

Reply:
This is in the defaults of the plugin and is set to `true` as would be main use case.